.cirrus.yml

```diff
@@ -333,6 +333,63 @@ task:
       on_failure:
         <<: *on_failure_ac
 
+    - name: Linux - Debian Bullseye - Meson - Test Extensions
+      env:
+        INSTALL_DIR: ${CIRRUS_WORKING_DIR}/meson-install
+        PATH: ${INSTALL_DIR}/bin:${PATH}
+
+      allow_failures: true
+      depends_on:
+        - Linux - Debian Bullseye - Meson
+
+      configure_script: |
+        su postgres <<-EOF
+          # Disable shadow=compatible-local until
+          # https://github.com/citusdata/pg_cron/issues/214 is fixed
+          export CFLAGS="${CFLAGS} -Wno-shadow=compatible-local"
```

Comment:
Maybe we could add that to pg_cron's section in test_extension instead? I think you can just add 'COPT=-Wno-shadow=compatible-local' to the 'make' invocation.

Reply:
I added 'COPT=-Wno-shadow=compatible-local' to 'test_pg_cron_script:' part, since adding it to 'make' command requires many changes on code(env variables are need to set). If issue persists or there will be more env variables to set, then I am planning to do these changes. Does it look good?

Reply:
You can just write `make install "COPT=-Wno-shadow=compatible-local'"`, no need for it to actually be an environment variable.